test/e2e/data/e2e_conf.yaml

```diff
@@ -12,11 +12,11 @@
 managementClusterName: capo-e2e
 
 images:
-- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller:nightly_master_20210310
+- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller:nightly_master_20210402
```

Comment:
one minor question: 

do we want to keep update this time to time ?

Reply:
Yes, will need a PR fein time to time until the release. Current estimation is June

Reply:
ok , thanks for the info~